src/ng/directive/form.js

```diff
@@ -131,6 +128,28 @@ function FormController(element, attrs, $scope, $animate, $interpolate) {
     if (control.$name) {
       form[control.$name] = control;
     }
+
+    control.$$parentForm = form;
+
+    forEach(control.$pending, function(value, name) {
+      form.$setValidity(name, undefined, control);
+    });
+
+    forEach(control.$error, function(value, name) {
+      form.$setValidity(name, false, control);
+    });
+
+    forEach(control.$$success, function(value, name) {
+      form.$setValidity(name, true, control);
+    });
+
+    if (control.$dirty && form.$pristine) {
+      form.$setDirty();
+    }
+
+    if (control.$submitted && !form.$submitted) {
+      form.$setSubmitted();
```

Comment:
Do we really want this ?
It is really a corner case, but if we already have unsubmitted form (with more fields/child-forms), then adding a submitted child-form shouldn't necessarily mark the form as submitted.
(The same holds for a submitted form and unsubmitted child-form being added.)

Neither way seems intuitive (so maybe it's better to leave the property untouched).